dapps/marketplace/src/pages/App.js

```diff
@@ -181,6 +181,10 @@ class App extends Component {
 export default withIsMobile(withWeb3(withCreatorConfig(withRouter(App))))
 
 require('react-styl')(`
+  html, body
```

Comment:
Do we need this? What's it for?

Reply:
Added that as fix to make scrolling/swiping a bit smooth on iOS webviews when using `position: fixed`. Since swipe to close is reverted, I can revert this too.